### DISCUSSION TITLE
Supply chain factor on openLCA

### DISCUSSION THREAD

**Discussion by aoifecalnan** [USEPA/supply-chain-factors#21] [Q&A] [answered]
Hi there, due to the update of this database, I was wondering if you now recommend using the Supply Chain Factors Dataset v1.1, instead of the USEEIO database that’s available through openLCA? Is there an intention to update the database which can be imported into openLCA with this new data?

**Comment by WesIngwersen** [comment, marked as answer]
Thanks for your question. The Supply Chain Factors is really the product intended for use with spend data for Scope 3 accounting.  See discussion #10 for more info on the differences between them.  Also please note that the most recent SEF dataset is v1.1.1, not v1.1. The differences are described in this announcement #6

**Reply by aoifecalnan** [reply]
Thank you for your reply, Wesley. If possible, will you be making openLCA users aware of the dataset changes? Having the Supply Chain Factors database available in openLCA would be really useful for users that want to look at other impacts beyond carbon.
